Codeinwp/neve
# Change deprecated function in jquery

### Summary
Change the deprecated function call in JQuery and also stop enqueuing that file if the user has the block editor.

### Will affect the visual aspect of the product
NO

### Test instructions
- Install the classic editor plugin and activate it
- Go to the new post
- Check if the "Enable Individual Content Width" shows the range slider when you activate it and if the range slider updates the input when you drag the range ( that's what that script does )
- Deactivate classic editor and check if [this script](https://prnt.sc/v306sl) is still there ( it shouldn't )


Closes #2173.
